### PR TITLE
titles must be capitalized

### PR DESCRIPTION
**Pull request by Posoroko**
## #851

Relevant [issue number](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #

## Please check the following

- [ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
added a new information to the translation guidelines:

"Titles must be consistently  capitalized in every languages"

### CODE REVIEW

File: docs/translation-guidelines.md [hunk 1]
```diff
@@ -4,7 +4,12 @@ Here, we try to determine a terminology for the game of Cuttle that will help fu
 
 
 ## The "Cuttle feel"
-Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal translations with the only goal of using the same word/expression as in the English version.  
+Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal translations with the only goal of using the same word/expression as in the English version.
+
+
+## Text formatting
+
+- **Capitalize Titles** :  Every word in a sentece used as a title must be capitalized.  (e.g. : title, goalTitle, cuttleTitle, etc )
```
Comment (Haviles04) [inline docs/translation-guidelines.md:12]: Sentence is spelled wrong here

**Comment by Haviles04** [comment]
LGTM

**Comment by Haviles04** [comment]
Is there a corresponding issue to this?

**Comment by Posoroko** [comment]
@Haviles04 I created an issue and edited the PR so it's linked to it.

File: docs/translation-guidelines.md [hunk 2]
```diff
@@ -4,7 +4,12 @@ Here, we try to determine a terminology for the game of Cuttle that will help fu
 
 
 ## The "Cuttle feel"
-Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal translations with the only goal of using the same word/expression as in the English version.  
+Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal translations with the only goal of using the same word/expression as in the English version.
+
+
+## Text formatting
+
+- **Capitalize Titles** :  Every word in a sentence used as a title must be capitalized.  (e.g. : title, goalTitle, cuttleTitle, etc )
```
Comment (Haviles04) [inline docs/translation-guidelines.md:12]: I feel like this is a little confusing, considering our asking for Capitalized but giving examples of camelCase

Comment (Posoroko) [reply docs/translation-guidelines.md:12]: yeah I see what you mean.  I'll make it clearer

File: docs/translation-guidelines.md [hunk 3]
```diff
@@ -4,7 +4,12 @@ Here, we try to determine a terminology for the game of Cuttle that will help fu
 
 
 ## The "Cuttle feel"
-Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal translations with the only goal of using the same word/expression as in the English version.  
+Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal translations with the only goal of using the same word/expression as in the English version.
+
+
+## Text formatting
+
+- **Capitalize Titles** :  Every word in a  title must be capitalized.  (e.g. : "counterTitle": "Chance To Counter" )
```
Comment (itsalaidbacklife) [inline docs/translation-guidelines.md:12]: Whitespace tweak
```suggestion
- **Capitalize Titles**: Every word in a title must be capitalized (e.g. : "counterTitle": "Chance To Counter" ).
```

**Review by itsalaidbacklife — APPROVED** [review]

**Pull request merged**